### PR TITLE
Live Report: Mark Findings as False Positive

### PR DESCRIPTION
**Pull request by filewalkwithme**
### This PR implements a button in the live report that allows users to mark findings as false positives.
alt text

### The button will display a modal pane where the user will be able to choose the new status for a finding.
alt text

### Also, there is an auxiliary tab where one can see the history of statuses changes for a finding:
alt text

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "api_url": "https://vulcan.example.com/api/v1/",
+  "api_url": "http://localhost:8081/api/v1/",
```
Comment (julianvilas) [inline src/config.json:2]: Is this change intended?

File: src/services/vulcan-api/.openapi-generator/VERSION [hunk 2]
```diff
@@ -1 +1 @@
-5.1.0-SNAPSHOT
\ No newline at end of file
+5.1.0
```
Comment (julianvilas) [inline src/services/vulcan-api/.openapi-generator/VERSION:1]: Is this generated automatically? It's missing the end-of-file return.

**Review by julianvilas — COMMENTED** [review]
Just two minor comments / questions.

File: src/config.json (hunk 1 above)
Comment (filewalkwithme) [reply src/config.json:2]: Good catch! This is my local setup. It shouldn't have being uploaded. I will restore it.

Comment (filewalkwithme) [reply src/config.json]: Done!

File: src/services/vulcan-api/.openapi-generator/VERSION (hunk 2 above)
Comment (filewalkwithme) [reply src/services/vulcan-api/.openapi-generator/VERSION:1]: Yes, it's auto-generated.  See https://github.com/adevinta/vulcan-ui/pull/2/files/70f1d7022f00898a525296b1ddd6060dda943be2#L10

**Review by julianvilas — APPROVED** [review]
